inc/t_cose/t_cose_sign1_sign.h

```diff
@@ -418,6 +419,7 @@ t_cose_sign1_encode_parameters(struct t_cose_sign1_sign_ctx *context,
  */
 static enum t_cose_err_t
 t_cose_sign1_encode_signature(struct t_cose_sign1_sign_ctx *context,
+                              struct q_useful_buf_c         payload,
```

Comment:
This interface can't be changed. Backwards compatibility needs to be provided here.